docs/web_ssr/set-up-and-usage/login-and-logout.md

```diff
@@ -24,23 +28,84 @@ window.mealz.user.loadWithExternalId(userID: string, forbidProfiling: boolean).s
   }
+}
+```
 
+### Log out
+
+When your user logs out, call `window.mealz.user.reset()` to disconnect the user from Mealz. Additionally, when the user logs out, you must generate a new authless ID and pass it in the headers in the future, as detailed in the [Pre-rendered Components](../main-features/pre-rendered-components) section.
+
+Just like you inform Mealz of the user's id, if your user is logged out (when Mealz scripts are loaded or after calling `window.mealz.user.reset()`), you will need to inform Mealz of the authless-id so it can remember the unlogged user:
+
+```ts
+window.mealz.user.loadWithAuthlessId(authlessId: string, forbidProfiling?: boolean);
+```
+
+It works similarly to the method `loadWithExternalId`, except you do not need to subscribe on it, as it does not do anything asynchronous on Mealz end.
+
+```ts
+// Example Setup
+export class Mealz {
   // Call this method from your app when the user logs out
   handleLogout() {
     window.mealz.user.reset();
+    this.handleAuthless()
+  }
+
+  // Call this method from your app during Mealz setup if the user is logged out
+  handleAuthless() {
+    // ...
+    // generate the new authless-id via the route /generate-authless-token
+    // ...
+    window.mealz.user.loadWithAuthlessId(authlessId, user.cookiesAccepted());
   }
 }
 ```
+
+:::info
+  Since the `loadWithExternalId` and `loadWithAuthlessId` carry the information of the profiling permission, if the permission changes while the scripts are active, you can call `window.mealz.user.updateForbidProfiling` to update the information (see [window.mealz](../customization/window-mealz#windowmealzuser) for method usage).
+:::
+
+## When Mealz scripts are not active
+
+If your user logs in when Mealz scripts are not active, there is the question of what to do the next time our components are displayed.
+
+If at this moment on startup you do the startup "normally" and pass to us the user-id, Mealz will treat the new user as-is and not as a continuation of the previous user. Which is fine, except if your website autorises unlogged carts and if the unlogged user had already pushed some recipes to cart.
+
+> **If that happens, the user will lose the recipes they had previously added to their cart !**
+
+To avoid this situation, there is a route you can call to inform Mealz that the unlogged user with a authless-id has logged and to transfer their current cart to their user-id:
+
+The route to **transfer the authless basket**:
+
+```
+GET http://MEALZ_SSR_API_URL/API_VERSION/basket/merge-authless-basket?authless_id=xxx&store_id=xxx
+```
+
+- Parameters :
+  - `authless_id: string`: **_(Mandatory)_** the authless-id of the user before they logged in
+  - `store_id: string`: **_(Mandatory)_** the id of the current store chosen by the user
+
+:::info
+  As the route transfers the basket of the authless-id to the user-id, you of course need to use the `Authorization: user_id` header when calling it and not the `Authless-id` header
```

Comment:
```suggestion
Since the route transfers the basket from the authless ID to the user ID, you need to use the `Authorization: user_id` header when calling it, instead of the `Authless-id` header.
```